db/schema.rb

```diff
@@ -169,9 +169,10 @@
     t.text "consumer_key", null: false
     t.text "shared_secret", null: false
     t.text "lms_link"
-    t.bigint "organization_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.bigint "organization_id"
```

Comment:
Why is this change happening? Does it need to be moved?

Reply:
I think that's just something that happened when regenerating the schema using `rake db:migrate` and adding another optional field, don't know why specifically though